### PR TITLE
Lint workflow, more contributors and more!

### LINKED ISSUE #8: Lint
https://github.com/scratch-contributors/scratch-contributors.github.io/issues/5#issuecomment-851955411

### PR DESCRIPTION
**Pull request by FunctionalMetatable**
Resolves #8

Adds a github action where it formats the repository's code. If the repository is owned by @scratch-contributors, it will use the @scratchcontributors-bot account to commit the changes. If the repository is **not** owned by @scratch-contributors, it will automatically use the push commiter's account to commit changes.

This pull request also fixes deploying from other branches (only if committing from the main branch the deploy action will work).

This also adds more contributors to the contributors page (I didn't put them in another branch because that would deploy it when I push a commit and this PR will fix that).

Some extra theme changes.

cc @leahcimto

### CODE REVIEW

**Comment by gosoccerboy5** [comment]
If you think this will work then I trust you

**Comment by gosoccerboy5** [comment]
I'm not good at all with yml, bots, etc. lol

**Review by gosoccerboy5 — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -29,8 +31,20 @@ jobs:
...
               cd src
               npm run generate
-         - name: Deploy to GitHub Pages
+              
+        
+         - name: Deploy to GitHub Pages (in main repository)
+           if: github.repository_owner == 'scratch-contributors'
            uses: peaceiris/actions-gh-pages@v3
            with:
               github_token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (leahcimto) [inline .github/workflows/deploy.yml:40]: use GH_PAT

**Review by leahcimto — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (FunctionalMetatable) [reply .github/workflows/deploy.yml:40]: we dont actually need it

Comment (FunctionalMetatable) [reply .github/workflows/deploy.yml:40]: note that you may have to give editing access to the bot

**Comment by FunctionalMetatable** [comment]
@leahcimto could you please give editing access to @scratchcontributors-bot?

**Comment by leahcimto** [comment]
It already has it

**Review by leahcimto — APPROVED** [review]
Looks good! Thanks for doing this

**Comment by FunctionalMetatable** [comment]
> It already has it

oh

**Comment by FunctionalMetatable** [comment]
> It already has it

[doesnt look like it does..](https://github.com/scratch-contributors/scratch-contributors.github.io/runs/2731316902)

**Comment by leahcimto** [comment]
Strange... just updated it

**Pull request merged**